Sources/SwiftTUI/View/Views/Stacks/ZStack.swift

```diff
@@ -0,0 +1,116 @@
+import Foundation
+
+public struct ZStack<Content: View>: View, Primitive, LayoutRoot {
+    public let content: Content
+    let alignment: Alignment
+
+    public init(alignment: Alignment = .center, @ViewBuilder _ content: () -> Content) {
+        self.content = content()
+        self.alignment = alignment
+    }
+
+    init(content: Content, alignment: Alignment = .center) {
+        self.content = content
+        self.alignment = alignment
+    }
+
+    static var size: Int? { 1 }
+
+    func loadData(node: Node) {
+        for i in 0 ..< node.children[0].size {
+            (node.control as! ZStackControl).addSubview(node.children[0].control(at: i), at: i)
```

Comment:
I noticed that the contents of the `ZStack` are drawn in the wrong order. I think that would be confusing for anyone trying to use it. Not sure what's the best way of dealing with this, we might be able to get away with changing the order in which any control draws its children, but that would require some testing if nothing else breaks.

Reply:
I think this will break `ZStack`s with dynamic content (e.g. with an `if` statement in it).

Reply:
That's true.

I validated the changes with what I thought was a moderately complex drop down menu. But in my example code the dynamic content was in the children of the `ZStack`.

I checked now and, as you said, adding an `if` directly to the `ZStack` content does not work.

Reply:
With the new set of changes, I observed dynamic content issues seem to be fixed.